engine-tests/src/test/java/org/terasology/persistence/serializers/VectorTypeSerializerTest.java

```diff
@@ -61,61 +50,17 @@ public void setup() {
     @Test
     public void testSerializationConstant() throws IOException {
         TestObject2 a = new TestObject2();
-        a.v1 = new org.joml.Vector3f(1.0f, 2.0f, 3.0f);
-        a.v2 = new org.joml.Vector4f(1.0f, 2.0f, 3.0f, 5.0f);
-        a.v3 = new org.joml.Vector2f(1.0f, 2.0f);
+        a.v1 = new Vector3f(1.0f, 2.0f, 3.0f);
+        a.v2 = new Vector4f(1.0f, 2.0f, 3.0f, 5.0f);
+        a.v3 = new Vector2f(1.0f, 2.0f);
         String data = gsonSerializer.toJson(a, new TypeInfo<TestObject2>() {
         });
 
         TestObject2 o = gsonSerializer.fromJson(data, new TypeInfo<TestObject2>() {
         });
-        assertEquals(o.v1, new org.joml.Vector3f(1.0f, 2.0f, 3.0f), .00001f);
-        assertEquals(o.v2, new org.joml.Vector4f(1.0f, 2.0f, 3.0f, 5.0f), .00001f);
-        assertEquals(o.v3, new org.joml.Vector2f(1.0f, 2.0f), .00001f);
-    }
-
-    @Test
-    public void testJsonSerializeRemapped() throws IOException {
-        TestObject a = new TestObject();
-        a.v1 = new Vector3f(11.5f, 13.15f, 3);
-        a.v2 = new Vector2f(12, 13f);
-        a.v3 = new Vector4f(12, 12.2f, 3f, 15.5f);
-        a.v11 = new org.joml.Vector3f(11.5f, 13.15f, 3);
-        a.v22 = new org.joml.Vector2f(12, 13f);
-        a.v33 = new org.joml.Vector4f(12, 12.2f, 3f, 15.5f);
-
-        String data = gsonSerializer.toJson(a, new TypeInfo<TestObject>() {
-        });
-
-        TestObject1 o = gsonSerializer.fromJson(data, new TypeInfo<TestObject1>() {
-        });
-
-        assertEquals(o.v1, new org.joml.Vector3f(11.5f, 13.15f, 3), .00001f);
-        assertEquals(o.v2, new org.joml.Vector2f(12f, 13f), .00001f);
-        assertEquals(o.v3, new org.joml.Vector4f(12, 12.2f, 3f, 15.5f), .00001f);
-
-    }
-
-    @Test
-    public void testProtobufSerializeRemapped() throws IOException {
```

Comment:
Why drop these serialization tests?

Reply:
they were for mapping between termath and joml. since we don't really have any strong backwards support it doesn't hurt to drop it.